builtin/providers/aws/resource_aws_iam_role.go

```diff
@@ -128,6 +139,13 @@ func resourceAwsIamRoleReadResult(d *schema.ResourceData, role *iam.Role) error
 	if err := d.Set("unique_id", role.RoleId); err != nil {
 		return err
 	}
+	policy, err := url.QueryUnescape(*role.AssumeRolePolicyDocument)
```

Comment:
I don't think we need to use a combination of `url.QueryUnescape` + `normalizePolicyDocument` - we should just be able to use the `normalizeJson` func already in terraform.